src/core/viz/expressions/basic/variable.js

```diff
@@ -29,13 +29,16 @@ import { checkString } from '../utils';
  */
 export default class Variable extends BaseExpression {
     constructor(name) {
-        checkString('name', 'name', 0, name);
+        checkString('variable', 'name', 0, name);
         if (name == '') {
             throw new Error('variable(): invalid parameter, zero-length string');
         }
         super({});
         this.name = name;
     }
+    _isFeatureDependent(){
+        return this.alias? this.alias._isFeatureDependent(): undefined;
```

Comment:
why not returning false?

Reply:
Because it is technically undefined, a variable may or may not be feature dependent until we know the aliased expression

Reply:
Done